code/game/objects/items/stacks/tiles/light.dm

```diff
@@ -20,7 +20,7 @@
 	else
 		return ..()
 
-/obj/item/stack/tile/light/place_tile(turf/open/T)
+/obj/item/stack/tile/light/place_tile(turf/open/target_plating, mob/user)
 	. = ..()
-	var/turf/open/floor/light/F = .
-	F?.state = state
+	var/turf/open/floor/light/floor = .
+	floor?.state = state
```

Comment:
why would floor be null here?

Reply:
If place tile is fired but failed e.g. runs out of mats,